package.json

```diff
@@ -36,6 +36,7 @@
     "start:prod": "yarn assets && yarn build:server && NODE_ENV=production yarn start",
     "start": "scripts/start.sh",
     "storybook": "concurrently --raw --kill-others 'yarn relay --watch' 'start-storybook --quiet -s ./public -p 9001'",
+    "sync-schema": "scripts/sync-schema-pull.sh",
```

Comment:
I was looking for this the other day and was wondering if it was all a dream -- ty 👍